recipe/meta.yaml

```
@@ -24,6 +24,10 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
```

We had the same duplication issue at ewah-bool-utils
```suggestion
    - cython                                 # [build_platform != target_platform]
```